<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ganerator map</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="mapWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>471</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>250</width>
      <height>600</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>PANEL OBSŁUGI</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pbWczytaj">
       <property name="text">
        <string>Wczytaj mapę</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbZapiszObraz">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Zapisz obraz mapy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbZapiszTxt">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Zapisz w formacie .txt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbUsun">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Usuń mapę</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbGeneruj">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Generuj mapę</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>            PARAMETRY</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lSzerokosc">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SZEROKOŚĆ: 100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hsSzerokosc">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lWysokosc">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>WYSOKOŚĆ: 100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hsWysokosc">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lWypelnienie">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>WYPEŁNIENIE MAPY: 50%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hsWypelnienie">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lZiarna">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>LICZBA ZIAREN: 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="hsLiczbaZiaren">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
